eloquent-js/chap15/q1.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Chapter 15 Question 1</title>
+</head>
+<body>
+    <p id="balloon">🎈</p>
+
+    <script>
+        const balloon = document.querySelector("#balloon");
+        const inflationPercentage = 10;
+        const maxBalloonSizeInPixels = 30;
+
+        window.addEventListener("keyup", checkIfToInflateBalloon);
+        window.addEventListener("keydown", checkIfToDeflateBalloon);
+
+        balloon.style.fontSize = "16px";
+
+        function checkIfToInflateBalloon(event) {
+            const balloonInflationKey = "ArrowUp";
+
+            if (event.key === balloonInflationKey) {
```

Comment:
Correct intention, but in this specific case not really necessary, because `event.key === "ArrowUp"` is clear enough